app/Http/Controllers/Auth/EmailVerificationNotificationController.php

```diff
@@ -14,7 +14,7 @@ class EmailVerificationNotificationController extends Controller
     public function store(Request $request): RedirectResponse
     {
         if ($request->user()->hasVerifiedEmail()) {
-            return redirect()->intended(route('dashboard', absolute: false));
+            return redirect()->intended(route('employee', absolute: false));
```

Comment:
please revert this to dashboard. we have implemented a dashboard for common purposes not only for employee

Reply:
Reverted all changes done to Dashboard... create a new issue to properly implement the dashboard